samples/subsys/ipc/ipm_imx/README.rst

```diff
@@ -0,0 +1,53 @@
+.. _ipm_imx:
+
+i.MX IPM example
+################
+
+Overview
+********
+A simple example that can be used with multicore i.MX SoC's containing the
```

Comment:
Make this a complete sentence, and change ``SoC's`` to plural spelling:

   This simple example can be used with multicore i.MX SoCs containing a
   Messaging Unit peripheral.